package.json

```diff
@@ -15,7 +15,7 @@
   "license": "MPL-2.0",
   "devDependencies": {
     "@types/dedent": "0.7.0",
-    "@types/node": "16.11.7",
+    "@types/node": "18.0.6",
```

Comment:
We actually use Node 16: https://github.com/tachiyomiorg/issue-moderator-action/blob/5e482873a3ccc54aa44865816fc42a8d7c24721a/action.yml#L49

Reply:
Ah, oops. Didn't know the types' version was tightly coupled with the Node one.